hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/table/HoodieFlinkCopyOnWriteTable.java

```diff
@@ -378,7 +377,7 @@ protected Iterator<List<WriteStatus>> handleUpdateInternal(HoodieMergeHandle<?,
       throw new HoodieUpsertException(
           "Error in finding the old file path at commit " + instantTime + " for fileId: " + fileId);
     } else {
-      FlinkMergeHelper.newInstance().runMerge(this, upsertHandle);
+      HoodieMergeHelper.newInstance().runMerge(this, upsertHandle);
```

Comment:
+1 on removing `FlinkMergeHelper` in a seprarete PR
Currently, there is another huge PR affecting `BaseMergeHelper` and its subclasses https://github.com/apache/hudi/pull/5830. 
The day will come when one of us will have to solve merge conflict with master branch.
I'd prefer to do it ASAP because no `FlinkMergeHelper` means no conflict:)